charts/kubescape-operator/Chart.yaml

```diff
@@ -33,3 +33,8 @@ maintainers:
   url: https://www.linkedin.com/in/matthias-bertschy-b427b815/
 
 home: https://www.armosec.io/
+
+dependencies:
```

Comment:
This should be conditional.
Look [here](https://github.com/armosec/kubecop/blob/main/chart/kubecop/Chart.yaml)

Reply:
This will not work because the condition must be a boolean and the capabilities are strings (enable/disable)
That is why we will install the crd chart by default.

Reply:
a bit out of scope but I think we should start using boolean instead of these "enable" "disable" strings... at the end it always translates to boolean values in the configmaps..